moveit_ros/moveit_servo/launch/servo_example.launch.py

```diff
@@ -144,6 +146,7 @@ def generate_launch_description():
             moveit_config.robot_description,
             moveit_config.robot_description_semantic,
             moveit_config.robot_description_kinematics,
+            low_pass_filter_coeff,
```

Comment:
How about we put this just after `servo_params` , feels easy to miss when looking through otherwise.

Reply:
good point, will do